authors/apps/authentication/tests/test_models.py

```diff
@@ -15,30 +15,29 @@ def setUp(self):
         """"
         Initializes all our test variables.
         """
-        self.userManager = UserManager()
 
     def test_create_user(self):
         """
         Checks whether the User manager class creates a user with a username, email and password
```

Comment:
change the comment too to reflect what you are testing

Reply:
Okay. Let me work on it.